CONTRIBUTING.md

```diff
@@ -110,29 +110,38 @@ VS Code
 
 2. To setup your development environment, first install [Docker](https://docs.docker.com/install/) and [Docker-Compose](https://docs.docker.com/compose/).
 
-> [!NOTE]\
-> If you are new to Docker, as an alternative activist recommends installing [Docker Desktop](https://docs.docker.com/desktop/). Docker Desktop comes with many Docker tools and a straightforward user interface.
+   > [!NOTE]\
+   > If you are new to Docker, as an alternative activist recommends installing [Docker Desktop](https://docs.docker.com/desktop/). Docker Desktop comes with many Docker tools and a straightforward user interface.
+
+3. [Fork](http://help.github.com/fork-a-repo/) the repo, clone your fork, and configure the remotes:
+
+   ```bash
+   # Clone your fork of the repo into the current directory
+   git clone https://github.com/<your-username>/activist.git
+   # Navigate to the newly cloned directory
+   cd activist
+   # Assign the original repo to a remote called "upstream"
```

Comment:
small nit - I know that @andrewtavis would've said something about ending the comments with periods `.`; so I'll mention it so he doesn't feel like he has to hihihi :laughing::smirk:

Reply:
Yea haha, I'll keep the final dots :)